WormsApp/MapperWrapper.h

```diff
@@ -1,18 +1,22 @@
-#ifndef MAPPERWRAPPER_H
-#define MAPPERWRAPPER_H
+#pragma once
 
 #include <unordered_map>
 #include <utility>
 #include<thread>
 
-template <typename KeyType, typename valueType> class MapperWrapper
+template <typename keyType, typename valueType>
+class MapperWrapper
 {
-    public:
-        void add(KeyType key, valueType&& value)
-        {
-        mapper.emplace(key, std::forward<std::thread>(value));
-        }
+public:
+    void add(keyType key, valueType&& value)
```

Comment:
This is great that you used lvalue reference here, because we actually want to move objects into our map, so it is the most appropriate type.